static/js/popovers.js

```diff
@@ -384,10 +384,19 @@ exports.hide_emoji_map_popover = function () {
     if (emoji_map_is_open) {
         $('.emoji_popover').css('display', 'none');
         $('.drag').css('display', 'none');
+        $("#new_message_content").focus();
         emoji_map_is_open = false;
     }
 };
-
+exports.show_emoji_map_popover = function () {
```

Comment:
Added  a function to show emoji popover. As there is one to hide the emoji popover.